SpriteBuilder/ccBuilder/ResourceManagerUtil.m

```diff
@@ -305,40 +316,40 @@ + (void)setAttributedTitle:(NSString*)fontName ofMenuItem:(NSMenuItem*)item {
 #pragma mark File icons
 
 + (NSImage*) thumbnailImageForResource:(RMResource*)res {
-    NSImage *image = [res previewForResolution:nil];
+
+    NSString* path = [res absoluteResolutionPath:nil];
     
-    if (![image isValid])
-    {
-        NSLog(@"Invalid Image");
+    CGFloat viewScale = [AppDelegate appDelegate].derivedViewScaleFactor;
+    
+    CGSize size = CGSizeMake(kRMImagePreviewSize*viewScale, kRMImagePreviewSize*viewScale);
+
+    NSURL *fileURL = [NSURL fileURLWithPath:path];
```

Comment:
Could be moved up to line 321 as well as the guard statement hereafter before anything else is done.